### PR TITLE
allow value scale to be reversed

### PR DESCRIPTION
**Pull request by peter-mouland**

### CODE REVIEW

File: src/scripts/axis/create.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ function timeScale(model, options, orientation) {
 
 function linearScale(model, options, orientation) {
     var range = getRange(model, orientation);
-    var domain = (isVertical(orientation)) ? model.dependentDomain.reverse() : model.dependentDomain;
+    var domain = (isVertical(orientation) && !model.y.reverse) ? model.dependentDomain.reverse() : model.dependentDomain;
```
Comment (radiodario) [inline src/scripts/axis/create.js]: This makes sense but i'd favour an if-else style, for clarity:

```js
var domain = model.dependentDomain;
if (isVertical(orientation) && !model.y.reverse) { 
  model.dependentDomain.reverse()
}
```

I'm not a big fan of inline ifs unless you're inside objects.

**Comment by radiodario** [comment]
Can we have a test or at least an example added the demo?

**Comment by radiodario** [comment]
anyway i had a look at the code and tried reversing the line chart and it works:
screen shot 2015-07-03 at 09 42 12

Are we planning to support reversed bars/columns? they look a bit funky currently:
screen shot 2015-07-03 at 09 43 18

**Comment by peter-mouland** [comment]
@radiodario refactored the code to follow a more api-like model.

i.e. o-charts users can now do 
```
var axis = oCharts.axis.number()
          .simple(true)
          .orient('left')
          .reverse(true)
          .scale(scale);
```

**Comment by peter-mouland** [comment]
#82 has been created to remind us about the bar charts

**Comment by radiodario** [comment]
Looks good, much more elegant solution

**Pull request closed without merging**